src/js/components/Pages.js

```diff
@@ -0,0 +1,28 @@
+/*global SEGMENT_KEY:true*/
+
+import React, { PropTypes } from 'react';
+import Header from './common/Header';
+import Footer from './common/Footer';
+import toastr from 'toastr';
+
+class Pages extends React.Component {
+  componentWillMount() {
+    toastr.options.positionClass = "toast-nav-top-right";
+  }
+
+  render () {
+    return (
+      <div>
+        <Header />
+        {this.props.children}
+        <Footer />
+      </div>
+    )
+  }
+}
+
+Pages.propTypes = {
+  children: PropTypes.object.isRequired
+}
```

Comment:
Proptype validation 👍 
Remind me to move us over to Facebook's Flow for prop validation instead of `React.PropTypes`